metashare/api/jobs.py

```diff
@@ -33,14 +33,15 @@ def _create_branches_on_github(*, user, repo_url, project, task, repo_root):
     if project.branch_name:
         project_branch_name = project.branch_name
     else:
-        prefix = get_cumulus_prefix(
-            repo_root=repo_root,
-            repo_name=repository.name,
-            repo_url=repo_url,
-            repo_owner=owner,
-            repo_branch=repository.default_branch,
-            repo_commit=repository.branch(repository.default_branch).latest_sha(),
-        )
+        with local_github_checkout(user, repo_url) as repo_root:
```

Comment:
Ugh, I don't like this but I see why it's necessary. If we did a full git checkout we could swap branches and minimize network activity, but I guess that would really just pre-load that not minimize.

Reply:
In practice this duplicate checkout should only happen on the very first org created for an entire project, so at least it's not every run.